script.js

```diff
@@ -85,12 +93,20 @@ function getBeerRec() {
 }
 
 function displayBeers(responseJson, targetedDiv) {
-    for (let i = 0; i < 3; i++) {
-        $(targetedDiv).append(`<div class="beers"><h3>Beer Recomendations based of your recipe....</h3>
-          <h4>${responseJson[i].name}</h4>
-          <img src="${responseJson[i].image_url}" alt="" width="20%">
-          <p>${responseJson[i].description}</p></div>`)
+    // we can't assume there will always be 3 or more results in responseJson
+    // so we can limit our loop to both our upper limit 3 and the length of results
+    let { length = 0 } = responseJson;
+    if (length > 0) {
+        for (let i = 0; i < 3 && i < length; i++) {
```

Comment:
If there are at least some results, we'll iterate up to at most 3 times, showing 1-3 beers that pair with the recipe.